README.md

```diff
@@ -15,7 +15,7 @@ This library implements SSDP as defined in the
 and uses HTTP over Multicast UDP (HTTPMU) as defined in the
 [IETF draft](https://tools.ietf.org/html/draft-goland-http-udp-01).
```

Comment:
While we're at it, it appears this would be the current URL:

```suggestion
[IETF draft](https://datatracker.ietf.org/doc/html/draft-goland-http-udp-01).
```

Reply:
okay should be the right URL now, thank you @clue 👍